### PR TITLE
Added buffer timespan plot instead of latency plot

### PR DESCRIPTION
**Pull request by maxsharabayko**
Depending on whether receiver or sender stats are provided, a timespan plot for either receiver or sender buffer is drawn.

### CODE REVIEW

File: plot_srt_stats.py [hunk 1]
```diff
@@ -193,7 +193,31 @@ def create_window_size_plot(source):
         lines,
         models.NumeralTickFormatter(format='0,0')
     )
+	
+def create_buf_timespan_plot(source, is_sender):
+    if is_sender and 'msSndBuf' in source.column_names:
+        lines = [linedesc('msSndBuf', '', 'blue')]
 
+        return create_plot(
+            'Sender buffer fullness',
```
Comment (mbakholdina) [inline plot_srt_stats.py]: Suggested change: 'Sender buffer fullness' -> 'Sender Buffer Fullness'

File: plot_srt_stats.py [hunk 2]
```diff
@@ -193,7 +193,31 @@ def create_window_size_plot(source):
...
+            'Timespan (ms)',
+            source,
+            lines
+        )
+
+    if  (not is_sender) and 'msRcvBuf' in source.column_names:
+        lines = [linedesc('msRcvBuf', '', 'blue')]
+
+        return create_plot(
+            'Receiver buffer fullness',
```
Comment (mbakholdina) [inline plot_srt_stats.py]: Suggested change: 'Receiver buffer fullness' -> 'Receiver Buffer Fullness'

File: plot_srt_stats.py [hunk 3]
```diff
@@ -393,7 +417,8 @@ def plot_graph(stats_filepath, is_sender, is_fec, export_png):
     plots['window_size'] = create_window_size_plot(source)
 
     # Latency
-    plots['latency'] = create_latency_plot(source)
+    #plots['latency'] = create_latency_plot(source)
```
Comment (mbakholdina) [inline plot_srt_stats.py:420]: Is the latency plot not actual anymore? The function `create_latency_plot` isn't used anymore.

**Review by mbakholdina — CHANGES_REQUESTED** [review]
Hi @maxsharabayko, I left a couple of comments. Please take a look.

File: plot_srt_stats.py (hunk 1 above)
Comment (maxsharabayko) [reply plot_srt_stats.py]:
```suggestion
            'Sender Buffer Fullness',
```

File: plot_srt_stats.py (hunk 2 above)
Comment (maxsharabayko) [reply plot_srt_stats.py]:
```suggestion
            'Receiver Buffer Fullness',
```

File: plot_srt_stats.py (hunk 3 above)
Comment (maxsharabayko) [reply plot_srt_stats.py:420]: The latency plot is not used at the moment, but the function remains in case it is needed in the future.

**Pull request merged**